developer/docs/help/guides/develop/write-keyboard-help-document.md.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Write a Keyman Keyboard help document
+---
+
+This is a guide to "what to put in a keyboard documentation", "how to start composing a PHP help file" from creating a file to a fully written documentation. 
+
+A keyboard help documentation (PHP help file) is a structured help web page about all of the information: 
+* Keyboard and language description
+* How to use the keyboard
+* Graphics for the Keyboard layouts for all platforms, and more. 
+
+The help documentation acts as an online help documentation on help.keyman.com, not to be confused with [welcome.htm and readme.htm](../distribute/tutorial/step-2). However, the help file can be a descriptive version of the `welcome.htm` file; see [here](https://help.keyman.com/developer/keyboards/phphelpfile).
+
+After planning and developing a keyboard in Keyman Developer, consider following the project's [folder and file structure](../../reference/file-layout), ideally the help file should be stored in an independent folder within the source folder. Therefore, create a folder called "**help**":
+
+```
+project
+|
+|
+|____source
+|    |      files
+|    |______help
+|           |
+|           |____keyboardname.php
+|
+|____other folders
+```
+
+In the `.php` file, include the main code:
+
+```php
+<?php 
+  $pagename = 'Project Name Keyboard Help';
+  $pagetitle = $pagename;
+  require_once('header.php');
+?>
+
+<!-- Write anything related to the keyboard here -->
+<div>
+  <h2>Desktop Keyboard layouts</h2>
+  <div id='osk' data-states='default shift rightalt rightalt-shift'>
+      <!-- By including this tag, the images of each layers for the 
+      On-screen keyboard will be placed here -->
+  </div>
+
+  <h2>Touch Keyboard layouts</h2>
+  <div id='osk-phone' data-states='default shift numeric symbol'>
+      <!-- By including this tag, the images of each layers for the 
+      Touch keyboard will be placed here -->
+  </div>
+</div>
+
+```
+
+Please feel free to adjust the `data-states` according to the keyboard layers; [see here](https://help.keyman.com/developer/keyboards/phphelpfile#toc-dynamically-constructed-keyboard-images).
+
+### Keep in mind
+
+* Some readers are visual learner, and they might be pleased to find a documentation that has more to the eyes than plain texts, check [Khmer (SIL) Keyboard Help](https://help.keyman.com/keyboard/sil_khmer/) for minimal design of keys (`<kbd>`) and one of the many methods to showcase key combinations.
+
+* Please avoid using this:
```

Comment:
Can you explain why to avoid the following? Maybe combine the paragraph in line 78 to here